guides/source/active_support_instrumentation.md

```diff
@@ -116,6 +116,7 @@ Action Controller
 | `:format`     | html/js/json/xml etc                                      |
 | `:method`     | HTTP request verb                                         |
 | `:path`       | Request path                                              |
+| `:request_id` | Request id                                                |
```

Comment:
I will also be in `process_action.action_controller`...

Reply:
True